ansible/roles/baselayout/vars/main.yml

```
@@ -18,7 +18,7 @@ sshd_service_name: "{{ sshd_service_map[os]|default(sshd_service_map[os|stripver
 
 ntp_service: {
   chrony: ['rhel8', 'rhel9', 'debian11'],
-  systemd: ['debian8', 'debian10', 'debian12', 'ubuntu']
+  systemd: ['debian8', 'debian12', 'ubuntu']
```

```suggestion
  systemd: ['debian12', 'ubuntu']
```

We may as well remove `debian8` (either as part of this PR or in a separate one).

---

SGTM